src/interfaces.ts

```diff
@@ -146,3 +146,19 @@ export interface RunnerOptions extends WorkerPoolOptions {
    */
   pgPool?: Pool;
 }
+
+export interface WorkerUtilsOptions {
```

Comment:
We should maybe rename this to "configuration" so that "configuration" and "options" remain distinct. If you can think of better words (or think they should be the other way around) please let me know :)